eng/common/pipelines/templates/steps/credscan.yml

```diff
@@ -8,6 +8,10 @@ steps:
 - pwsh: |
     if ("$(Build.Reason)" -eq 'PullRequest') {
       $targetBranch = "origin/$(System.PullRequest.TargetBranch)" -replace "refs/heads/"
+      git config core.quotepath off
+      # Disable the unix path encoding.
+      git config --unset i18n.logoutputencoding
+
       $changedFiles = git diff $targetBranch HEAD --name-only --diff-filter=d
```

Comment:
```suggestion
      git config core.quotepath off
      # Disable the unix path encoding.
      git config --unset i18n.logoutputencoding

      $changedFiles = git diff -c core.quotepath=off -c i18n.logoutputencoding=utf-8 $targetBranch HEAD --name-only --diff-filter=d
```

Reply:
I generally like to avoid changing global git settings so lets scope these config changes just to the one call.